src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANGE_VALUES = 10;
+    private final Random random = new Random();
+    private final ColorSupplier supplierColor = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FigureTypes.values().length);
+        switch (figureNumber) {
+            case 0:
+                return createCircle();
+            case 1:
+                return createIsoscelesTrapezoid();
+            case 2:
+                return createRectangle();
+            case 3:
+                return createRightTriangle();
+            default: break;
+        }
+        return createSquare();
+    }
+
+    public Figure getDefaultFigure() {
+        Circle circle = new Circle();
+        circle.setRadius(10);
+        circle.setColor(Color.WHITE.name());
+        circle.getInfoFigure();

```

Comment:
```suggestion
        circle.setRadius(DEFAULT_RADIUS);
        circle.setColor(DEFAULT_COLOR);
```
this method should not be called here

lets make constants for default values